workspaces/sonarqube/.changeset/strong-forks-smash.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage-community/plugin-sonarqube': minor
+---
+
+- Fixed handling of wrong component keys in getFindingSummaries
+- Support all entity kinds in the tabular view
+- Make the tabular view more compact and sortable by gateway/last analyzed
+- Improve translations
+- Added `SonarQubeRelatedEntitiesOverview` to show SonarQube results for related entites
```

Comment:
There's a lot of changes here, some of these could be just `patch` updates in separate PRs. 🤔

Reply:
Yeah, having looked this over I think breaking this into smaller chunks would be beneficial and easier to review. Sorry, I didn't realize this was the extent of the changes you had in mind. I think these all make sense to me I just think there's a lot moving in this single PR for me to confidently review it.